docs/en/observability/cloud-monitoring/aws/monitor-aws-agent.asciidoc

```diff
@@ -1,6 +1,10 @@
 [[monitor-aws-elastic-agent]]
 == Monitor Amazon Web Services (AWS) with {agent}
 
+++++
+<titleabbrev>Monitor {aws} with {agent}</titleabbrev>
+++++
+
 ****
 **New to Elastic?** Follow the steps in our {estc-welcome}/getting-started-observability.html[getting started guide] instead
```

Comment:
This is the only tutorial that has this admonition. Should we remove it? Add it to all pages?

Reply:
Removed it 👍